src/Bundle/Resources/config/services.xml

```diff
@@ -13,12 +13,17 @@
             <argument type="service" id="serializer.mapping.class_discriminator_resolver" on-invalid="ignore" />
         </service>
 
+        <service id="dunglas_doctrine_json_odm.normalizer.test" class="Dunglas\DoctrineJsonOdm\Tests\ProductNormalizerTest" public="false">
```

Comment:
These services are going to be used in production. You cannot add a test normalizer here :)

Reply:
I understand and I wrote about this originally, but I dont know how add custom normalizer in test properly without of `services.yaml`. So I did this hardcoding only for demo purposes and this must be removed then.

Reply:
This service definition could be added directly in https://github.com/dunglas/doctrine-json-odm/blob/master/tests/Fixtures/AppKernel.php#L43